bin/create-initial-roles-and-users.sh

```diff
@@ -51,3 +55,7 @@ cd "$parent_path"
 ./idam-create-caseworker.sh citizen befta.citizen.3@gmail.com Pa55word11
 ./idam-create-caseworker.sh caseworker,caseworker-befta_jurisdiction_3 befta.caseworker.3@gmail.com Pa55word11
 ./idam-create-caseworker.sh caseworker,caseworker-befta_jurisdiction_1 befta.caseworker.1.noprofile@gmail.com Pa55word11 testsurname testfirstname
+./idam-create-caseworker.sh caseworker,caseworker-befta_master master.caseworker@gmail.com Pa55word11 testsurname testfirstname
+./idam-create-caseworker.sh caseworker,caseworker-befta_master,caseworker-befta_master-solicitor_1 master.solicitor.1@gmail.com Pa55word11 testsurname testfirstname
```

Comment:
Could the user first names or surnames be differentiated to denote the user for convenience on UI, diagnostics, etc.